web/settings/controller_test.go

```diff
@@ -28,7 +28,7 @@ import (
 	_ "github.com/cozy/cozy-stack/worker/mails"
```

Comment:
Context: https://github.com/cozy/cozy-stack/pull/3988#discussion_r1234043674

Context 2: I wanted to write the tests for the `HTTPHandler` (and only this) but the file `setting_test.go` was already used with tests tests on more elements. As we didn't agree (yet) on what is unit/integration/E2E tests and where we use what we decided to name it `controller_test.go`. 

End: After some checks it will be very hard at the moment to make http tests due to the way the session is handled by global functions calling directly the db so I will revert the commit doing the renaming.